pom.xml

```diff
@@ -1899,25 +1885,21 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-shade-plugin</artifactId>
-                    <version>${maven.plugin.shade.version}</version>
                 </plugin>
 
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-jar-plugin</artifactId>
-                    <version>${maven.plugin.jar.version}</version>
                 </plugin>
```

Comment:
we can remove the simple definition in the `dependencyManagement` block then.

Reply:
Yes, agree.